tests/test_routes.py

```diff
@@ -324,6 +324,60 @@ def test_create_with_invalid_content_type(self):
         )
         self.assertEqual(response.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
 
+    def test_create_with_invalid_can_update_status_enum(self):
```

Comment:
I'm going to have to rebase this file off mainline since changes have been merged in just now

Reply:
Will do so later today